go get -u -d github.com/k8sp/sextant 并不会吧其他依赖的go代码下载下来

按照[Readme.md](https://github.com/k8sp/sextant/blob/master/README.md) 中的说明，执行
go get -u -d github.com/k8sp/sextant
``` shell
[liangjiameng@mars sextant]$ go get -u -d github.com/k8sp/sextant
package github.com/k8sp/sextant: no buildable Go source files in /work/golang/src/github.com/k8sp/sextant
```
进入 /work/golang/src/github.com/k8sp/sextant目录之后，执行：

``` shell
[liangjiameng@mars sextant]$ ./bsroot.sh cloud-config-server/template/cluster-desc.yml /bsroot/
/bsroot/ already exists. Overwrite without removing it.
Using bootstrapper server IP 10.10.14.253
Checking prerequisites ... Done
Downloading syslinux ... Done
Importing CoreOS signing key ... Done
Downloading CoreOS PXE vmlinuz image ... Done
Downloading CoreOS PXE CPIO image ... Done
Generating pxelinux.cfg ... Done
Generating Docker registry config file ... Done
Downloading and kubelet and kubectl of release v1.3.6 ... Done
Downloading setup-network-environment file ... Done
Copying cloud-config template and cluster-desc.yml ... Done
Copying bsroot_lib.bash ... Done
Generating install.sh ... Done
Checking new CoreOS version ... Done
Updating CoreOS images ... Done
Cross-compiling Sextant Go programs ... /work/golang/src/github.com/k8sp/sextant/cloud-config-server/server.go:24:2: cannot find package "github.com/gorilla/mux" in any of:
	/usr/local/go/src/github.com/gorilla/mux (from $GOROOT)
	/work/golang/src/github.com/gorilla/mux (from $GOPATH)
/work/golang/src/github.com/k8sp/sextant/cloud-config-server/cache/cache.go:9:2: cannot find package "github.com/topicai/candy" in any of:
	/usr/local/go/src/github.com/topicai/candy (from $GOROOT)
	/work/golang/src/github.com/topicai/candy (from $GOPATH)
/work/golang/src/github.com/k8sp/sextant/clusterdesc/linux_distro.go:8:2: cannot find package "github.com/wangkuiyi/sh" in any of:
	/usr/local/go/src/github.com/wangkuiyi/sh (from $GOROOT)
	/work/golang/src/github.com/wangkuiyi/sh (from $GOPATH)
/work/golang/src/github.com/k8sp/sextant/cloud-config-server/server.go:30:2: cannot find package "gopkg.in/yaml.v2" in any of:
	/usr/local/go/src/gopkg.in/yaml.v2 (from $GOROOT)
	/work/golang/src/gopkg.in/yaml.v2 (from $GOPATH)
Failed
```

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ bootstrapper需要运行在一台服务器上(以下称bootstrapper server)，�
 编辑cloud-config-server/template/cluster-desc.sample.yaml文件完成配置
 然后下载bootstrapper用到的文件到/bsroot目录下
 ```
-go get -u -d github.com/k8sp/sextant
+go get -u -d github.com/k8sp/sextant/...
 cd $GOPATH/src/github.com/k8sp/sextant
 vim cloud-config-server/template/cluster-desc.sample.yaml
 ./bsroot.sh cloud-config-server/template/cluster-desc.sample.yaml

